### ISSUE TITLE
GetComicInfo fails when CV description contains link missing period

### ISSUE THREAD

**Issue by PCurd** [evilhero/mylar#2164]
I THINK that's the cause... debug trace below but I'll explain my findings first.

When Mylar parses the CV description on 4050-51026 it finds three links, on the third it falls over because a None is being passed into re.sub - this seems to be because the title of the Next issue ends in a ? so no period is added to the end of the text by CV. The description ends:
```
Previous: Vol. 15: Invincible: Get Smart.
Next: Vol. 17: Invincible: What's Happening?
```

The ? in Vol 17 is part of the a tag: `<a href="/invincible-whats-happening/4050-55873/" data-ref-id="4050-55873">Invincible: What's Happening?</a>`

I think it should be easy to fix, with a check if issuerun is None on line 387. but my python isn't great and I don't know if there are other side effects from the None hanging around.

The debug log shows:
```
09-Jan-2019 10:36:43 - DEBUG :: mylar.GetComicInfo.226 : Thread-12 : number of issues I counted: 1
09-Jan-2019 10:36:43 - DEBUG :: mylar.GetComicInfo.227 : Thread-12 : number of issues CV says it has: 1
09-Jan-2019 10:36:43 - DEBUG :: mylar.GetComicInfo.233 : Thread-12 : vari is set to: no
09-Jan-2019 10:36:43 - ERROR :: mylar.excepthook.315 : Thread-12 : Uncaught exception: Traceback (most recent call last):
  File "D:\Mylar\mylar\mylar\logger.py", line 337, in new_run
    old_run(*args, **kwargs)
  File "c:\python27\lib\threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "D:\Mylar\mylar\mylar\importer.py", line 104, in addComictoDB
    comic = cv.getComic(comicid, 'comic')
  File "D:\Mylar\mylar\mylar\cv.py", line 156, in getComic
    return GetComicInfo(comicid, dom)
  File "D:\Mylar\mylar\mylar\cv.py", line 387, in GetComicInfo
    lines = re.sub("[^0-9]", ' ', issuerun).strip().split(' ')
  File "c:\python27\lib\re.py", line 155, in sub
    return _compile(pattern, flags).sub(repl, string, count)
TypeError: expected string or buffer

Traceback (most recent call last):
  File "D:\Mylar\mylar\mylar\logger.py", line 337, in new_run
    old_run(*args, **kwargs)
  File "c:\python27\lib\threading.py", line 754, in run
    self.__target(*self.__args, **self.__kwargs)
  File "D:\Mylar\mylar\mylar\importer.py", line 104, in addComictoDB
    comic = cv.getComic(comicid, 'comic')
  File "D:\Mylar\mylar\mylar\cv.py", line 156, in getComic
    return GetComicInfo(comicid, dom)
  File "D:\Mylar\mylar\mylar\cv.py", line 387, in GetComicInfo
    lines = re.sub("[^0-9]", ' ', issuerun).strip().split(' ')
  File "c:\python27\lib\re.py", line 155, in sub
    return _compile(pattern, flags).sub(repl, string, count)
TypeError: expected string or buffer
```

**Comment by evilhero** [comment]
It's actually due to the bullet form notes they have in the description along with the CV links. When Mylar parses the description it looks at all the text and tries to remove elements that don't belong to the current volume. In this case, each bullet point is being considered as more than one separate element and it thinks they are all part of the current volume. 

This is all so that it can nicely display the relevant collection information on the series detail page (under the Edition field), along with a hyperlink to any series that are already in your watchlist and exist in the collected edition. So yeah, I like to make more work - but the end result eventually will be that hopefully Mylar will be able to know what's inside each TPB/GN and thereby mark any issues you may already have separately as well as collectively.

Fix is up in development (625a36a) now for this issue topic.

**Comment by PCurd** [comment]
Excellent, thank you for fixing so fast! 

As an aside, the "collects: xx" feature is REALLY nice, it's worth the effort to me I think!
